### PR TITLE
fix: pizza-facts missing link to markdown file

### PR DESCRIPTION
**Pull request by thititongumpun**
## Description
This PR fixes a hyperlink to pizza-facts.md file in README.md caused a 404 error when accessed to the link due to the repo changed to [pizza-facts-&-trivia.md](https://github.com/open-sauced/pizza-verse/blob/main/pizza-facts-%26-trivia.md). This commit addresses the issue by renaming pizza-facts.md to "pizza-facts-&-trivia.md" for match the correct filename. The issue was identified during read pizza-related content in README.md. Also added another pizza fun facts content.

## What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [x] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Added tests?

- [ ] 👍 yes
- [ ] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [x] 📜 README.md
- [ ] 📓 docs.opensauced.pizza
- [ ] 🍕 dev.to/opensauced
- [ ] 📕 storybook
- [ ] 🙅 no documentation needed

### CODE REVIEW

**Comment by CBID2** [comment]
Hi @thititongumpun. There are some sections in your pull request form that are missing information. Can you add them please?

**Comment by thititongumpun** [comment]
Sure. I add some fun facts.

**Comment by CBID2** [comment]
> Sure. I add some fun facts.

To clarify @thititongumpun,  I mean fill out the form in its entirety. I recommend reading [this blog post](https://dev.to/opensauced/writing-your-first-pull-request-tips-best-practices-and-ai-powered-tools-for-success-3bg9) to help you. Once you make the changes, tag me again

**Comment by thititongumpun** [comment]
@CBID2 Sorry for miss understood. I made the changes.

File: pizza-facts-&-trivia.md [hunk 1]
```diff
@@ -7,5 +7,7 @@ We've categorized the facts into different sections to help explore the diverse
 ## Pizza Fun Facts
 
 1. A pizza was delivered in space 🚀 and Russian cosmonaut, Yuri Usachov, had the honor of being the first person to receive a pizza delivery while in orbit.
+2. The most popular pizza 🍕 topping is pepperoni, It’s not to hard to figure this one out, but now we have the numbers to prove it! In a HarrisPoll® taken in January of 2016, pepperoni was found to be the most popular topping followed by Sausage at number two.
```
Comment (CBID2) [inline pizza-facts-&-trivia.md:10]: [typo] 
```suggestion
2. The most popular pizza 🍕 topping is pepperoni. It’s not to hard to figure this one out, but now we have the numbers to prove it! In a HarrisPoll® taken in January of 2016, pepperoni was found to be the most popular topping followed by Sausage at number two.
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Looks great @thititongumpun! :) I just pointed out a typo that needs fixing

**Comment by thititongumpun** [comment]
@CBID2 Fixed. Thanks.

**Review by CBID2 — APPROVED** [review]
Well done @thititongumpun! :)

**Comment by CBID2** [comment]
Yay you just made your first contribution @thititongumpun! :) If you want to do more updates on OpenSauced, join our [Discord](https://discord.com/invite/U2peSNf23P) community! :) We'd love to see more of your ideas! :)

**Pull request merged**